libcontainer/standard_init_linux.go

```diff
@@ -187,6 +188,10 @@ func (l *linuxStandardInit) Init() error {
 			return newSystemErrorWithCause(err, "init seccomp")
 		}
 	}
+
+	logPipe, _ := (logrus.StandardLogger().Out).(*(os.File))
```

Comment:
Two things:

* I'm not sure it's necessary to do `(a).(*os.File)` -- I think `a.(*os.File)` will work just as fine (and `gofmt` should complain at you.

* You shouldn't ignore the `ok` value here. Even if it is impossible to occur, you should either make it panic or you should check that `logPipe != nil` (which is the error case -- because that would also panic).

* Also we should probably set `logPipe` to be `O_CLOEXEC` which would remove the requirement to close it like this.